src/baldr/geojson.cc

```diff
@@ -19,14 +19,13 @@ template <class coord_t>
 MapPtr to_geojson(const typename midgard::GriddedData<coord_t>::contours_t& grid_contours, bool polygons, const std::vector<std::string>& colors) {
   //for each contour interval
   int i = 0;
-  auto color_itr = colors.cbegin();
+  auto color_itr = colors.crbegin();
```

Comment:
just change the colors to a map of time to color and pass that instead right? then its not even an iterator its just a `color_map.find(interval.first)` and if its not in the map then we use the on the fly color derived from `i`. wont that work and be fool proof. sorry for not doing this the first go round...